core/src/services/azdls/lister.rs

```diff
@@ -136,6 +160,283 @@ struct Path {
     name: String,
 }
 
+pub(super) fn generate_continuation_from_start_after(start_after: &str) -> String {
```

Comment:
It's better to provide links to references so we know where that logic comes from.